src/workloads/query/TimeseriesTsbsOptimizations.yml

```diff
@@ -104,6 +104,60 @@ Actors:
                     },
                   ]
 
+  # Bucket-level rewrites to use the time field cannot be done due to the field not always being
+  # a true max. However, these queries will be eligible to use block processing.
+  - Name: BucketLevelRewriteGroupOnMetaMaxTime
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [5]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $group:
+                        {
+                          _id: "$tags.hostname",
+                          max_time: { $max: "$time" },
+                        },
+                    },
+                  ]
+
+  - Name: BucketLevelRewriteGroupOnMetaMinTime
+    Type: CrudActor
+    Database: *database
+    Threads: 10
+    Phases:
+      OnlyActiveInPhases:
+        Active: [7]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: *repeat
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  [
+                    {
+                      $group:
+                        {
+                          _id: "$tags.hostname",
+                          min_time: { $min: "$time" },
+                        },
+                    },
+                  ]
+
```

Comment:
The other query we talked about `{$group: {_id: time, a: {$min/max: meta}}}` wasn't affected, since it can't use bucket-level rewrites anyway.